onnxruntime/core/optimizer/qdq_transformer/selectors_actions/qdq_selectors.cc

```diff
@@ -51,14 +51,6 @@ bool NodeGroupSelector::CheckQDQNodes(const GraphViewer& graph_viewer, const Nod
     return false;
   }
 
-  auto does_node_produce_graph_output = [&graph_viewer](const Node* node_ptr) {
```

Comment:
revert this, CPU EP could benefit from this check. it doesn't deal with NodeUnits yet